buildvertical.targets

```diff
@@ -28,8 +28,8 @@
         <AdditionalProperties>Configuration=%(Identity);%(_projectBuildConfigurations.AdditionalProperties)</AdditionalProperties>
       </_projectBuildConfigurations>
 
-      <!-- transform back to project -->
-      <_projectWithConfiguration Include="@(_projectBuildConfigurations->'%(OriginalItemSpec)')" />
+      <!-- transform back to project, we need to ignore negative build configurations that start with _ -->
```

Comment:
Can we call them "placeholder configurations" instead of negative so that our "_" marker might make a little more sense.

Reply:
makes sense.